### PR TITLE
CRS-1129 update node version and orbs

### PR DESCRIPTION
**Pull request by joelstobart-moj**

### CODE REVIEW

**Review by yasinmustafa — CHANGES_REQUESTED** [review]
LGTM - except for the good question from @smcveigh941 and Ive added a comment about the cron timings

File: .circleci/config.yml [hunk 1]
```diff
@@ -198,7 +199,7 @@ workflows:
   security:
     triggers:
       - schedule:
-          cron: "5 3 * * 1-5"
+          cron: "30 5 * * 1-5"
```
Comment (yasinmustafa) [inline .circleci/config.yml:202]: I think these cron timings should remain as they were? I believe all projects have different times,

File: .circleci/config.yml [hunk 2]
```diff
@@ -223,7 +224,7 @@ workflows:
   security-weekly:
     triggers:
       - schedule:
-          cron: "41 4 * * 1"
+          cron: "0 5 * * 1"
```
Comment (yasinmustafa) [inline .circleci/config.yml:227]: I think these cron timings should remain as they were? I believe all projects have different times,

File: .circleci/config.yml [hunk 3]
```diff
@@ -148,7 +148,7 @@ workflows:
             - build
       - hmpps/helm_lint:
           name: helm_lint
-      - hmpps/build_docker:
+      - hmpps/build_multiplatform_docker:
```
[the remark this answers is not in the document]
Comment (joelstobart-moj) [reply .circleci/config.yml:151]: I was following the template project. Are the template project doing it wrong https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/.circleci/config.yml

File: .circleci/config.yml (hunk 1 above)
Comment (joelstobart-moj) [reply .circleci/config.yml:202]: I agree - resolved in the subsequent commit

File: .circleci/config.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (joelstobart-moj) [reply .circleci/config.yml]: I think that projects should use the best defaults or provide alternatives.

**Review by yasinmustafa — APPROVED** [review]
LGTM

**Pull request merged**